frontend/ui/uimanager.lua

```diff
@@ -124,6 +124,18 @@ function UIManager:setIgnoreTouchInput(state)
     InputContainer:setIgnoreTouchInput(state)
 end
 
+function UIManager:setSilentMode()
+    self.silent_mode = true
+end
+
+function UIManager:resetSilentMode()
+    self.silent_mode = nil
+end
+
+function UIManager:isSilentMode()
```

Comment:
isInSilentMode() would sound better ?
Pinging @NiLuJe to approve these namings or suggest some alternative.
Fine with me with how it ended up.

Reply:
Feels a bit ugly to have UIManager knows about (having their names in uimanager.lua) the classes that can be silenced. (+ the possible require loop if UIManager would require(infomessage) which itself require(uimanager) . If it's just their name as a string, it's even uglier :)).
Feels better to have the flag in UIManager, and classes announcing they support silent mode with a property.

So, I guess you're fine with the name "silent mode" ? What about isSilentMode() vs is***In***SilentMode() ?